bin/local_storage.rb

```diff
@@ -18,6 +20,16 @@ def load_data(app)
         new_book.add_label(new_label) if new_label
         app.books << new_book
       end
+
+      load_from_file(games_file).map do |game|
+        new_game = Game.from_hash(game)
+        app.games << new_game
+      end
+
+      load_from_file(authors_file).map do |author|
+        new_author = Author.from_hash(author)
+        app.authors << new_author
+      end
```

Comment:
Please, notice that you are not creating the associations between recovered games and authors, take a look at lines 15 until 22 to get an idea about how to do it.

For that, you first have to recover the authors, then recover games and add the proper author to each game.